README.md

```diff
@@ -84,10 +86,15 @@ Supported Cars
 | Honda     | Pilot 2019                    | All               | openpilot        | 25mph<sup>5</sup>  | 12mph        |
 | Honda     | Ridgeline 2017-19             | Honda Sensing     | openpilot        | 25mph<sup>5</sup>  | 12mph        |
 | Hyundai   | Elantra 2017-19<sup>1</sup>   | SCC + LKAS        | Stock            | 19mph              | 34mph        |
+| Hyundai   | Elantra GT 2017-19<sup>1</sup>| All               | Stock            | 0mph               | 30mph        |
 | Hyundai   | Genesis 2018<sup>1</sup>      | All               | Stock            | 19mph              | 34mph        |
+| Hyundai   | Ioniq 2017<sup>1</sup>        | All               | Stock            | 0mph               | 34mph        |
```

Comment:
Base Ioniq in the USA doesn't have SCC, requires Limited trim. I think all have LKAS. Probably should be changed to `SCC + LKAS`

Also need to add the line for `Ioniq 2019 Electric`, with `SCC + LKAS`

Reply:
merge